final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import xml.etree.ElementTree as ET
+import requests
+import json
+import logging
+import sys
+
+import rss_reader.ClassNews as ClassNews
+import rss_reader.CSVEntities as CSVEntities
+
+VERSION = 1.1
+
+
+def args_parser(args):
+    # Parse our arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument('source', nargs='?', help="RSS URL")
+    parser.add_argument('--version', action='store_true', help='Print version info')
+    parser.add_argument('--json', action='store_true', help='Print result as JSON in stdout')
+    parser.add_argument('--verbose', action='store_true', help='Outputs verbose status messages')
+    parser.add_argument('--limit', type=int, help='Limit news topics if this parameter provided')
+    parser.add_argument('--date', type=str, help='Date for selecting topics')
+
+    res_args = parser.parse_args(args)
+    return res_args
+
+
+def get_dict_from_xml(rss_request, limit):
+    main_title = ''
+    root = ET.fromstring(rss_request.content)
+
+    # Here we get title of api
+    for channel_info in root.iter('channel'):
+        for item in channel_info:
+            if item.tag == 'title':
+                main_title = item.text
+
+    # Here we have the dictionary of articles
+    res_dict_articles = ClassNews.xml_arguments_for_class(root, limit)
+    return res_dict_articles, main_title
+
+
+def get_request(args_source, timeout=None):
+
+    logging.info('Start parsing')
+    rss_request = requests.get(args_source, timeout=timeout)
+
+    # Check status code
+    status_code = rss_request.status_code
+    logging.info("Status code {}".format(status_code))
+    # if status_code == 404:
+    #     raise requests.exceptions.HTTPError
```

Comment:
Please avoid leaving commented-out code in the pushed commits

Reply:
overlooked, will fix